### PR TITLE
Q1 2022 roadmap: Feb - Apr

### PR DESCRIPTION
**Pull request by maryelizbeth**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTS=TestAccXXX PKG=ec2

...
```

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,74 +1,121 @@
...
 
 Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/hashicorp/terraform-provider-aws/milestone/138).
 
-This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
+This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur.
 
-In the period spanning August to October 2021, 573 Pull Requests were opened in the provider and 465 were merged, adding support for:
+In the period spanning November 2021 to Janury 2022, 627 Pull Requests were opened in the provider and 506 were merged, adding support for:
```
Comment (maryelizbeth) [inline ROADMAP.md:11]: @breathingdust these are stats as of today, will need to be updated end of week!

File: ROADMAP.md [hunk 2]
```diff
@@ -1,74 +1,121 @@
...
+
+Issue: [#20853](https://github.com/hashicorp/terraform-provider-aws/issues/20853)
+
+_[Amazon OpenSearch Service](https://aws.amazon.com/opensearch-service/) is a distributed, open-source search and analytics suite used for a broad set of use cases like real-time application monitoring, log analytics, and website search_
+
+Support for Amazon OpenSearch Service will include:
+
+New Resource(s):
+
+- `aws_elasticsearch_domain`
```
Comment (maryelizbeth) [inline ROADMAP.md:65]: Confirmation on this set of resources @breathingdust

File: ROADMAP.md [hunk 3]
```diff
@@ -1,74 +1,121 @@
...
+
+Issue: [#16789](https://github.com/hashicorp/terraform-provider-aws/issues/16789)
+
+_[Amazon Managed Grafana](https://aws.amazon.com/grafana) is a fully managed service for open source Grafana that enables you to query, visualize, alert on and understand your metrics._
+
+Support for Amazon Managed Grafana will include:
+
+New Resource(s):
+
+-
```
Comment (maryelizbeth) [inline ROADMAP.md:78]: need resource list here @breathingdust

File: ROADMAP.md (hunk 2 above)
Comment (breathingdust) [reply ROADMAP.md]: Might leave this out, ideally we will be able to use the same resources but we need to dig in a little further

**Pull request merged**